autoload/leetcode.py

```diff
@@ -17,17 +17,18 @@
     vim = None
 
 
-LC_BASE = 'https://leetcode.com'
-LC_LOGIN = 'https://leetcode.com/accounts/login/'
-LC_GRAPHQL = 'https://leetcode.com/graphql'
-LC_CATEGORY_PROBLEMS = 'https://leetcode.com/api/problems/{category}'
-LC_PROBLEM = 'https://leetcode.com/problems/{slug}/description'
-LC_TEST = 'https://leetcode.com/problems/{slug}/interpret_solution/'
-LC_SUBMIT = 'https://leetcode.com/problems/{slug}/submit/'
-LC_SUBMISSIONS = 'https://leetcode.com/api/submissions/{slug}'
-LC_SUBMISSION = 'https://leetcode.com/submissions/detail/{submission}/'
-LC_CHECK = 'https://leetcode.com/submissions/detail/{submission}/check/'
-LC_PROBLEM_SET_ALL = 'https://leetcode.com/problemset/all/'
+LC_BASE = ''
```

Comment:
Hi, IMHO there could be a better way to initialize the constants other than deliberately calling a function, since it is not intuitive to call a module function to initialize the module itself.

Since, according to the pull request, the only difference between the two regions is the base URL, a compromise could be defining `LC_BASE` via an environment variable like `LEETCODE_BASE_URL=https://leetcode.com`, and then defining all the other constants based on `LC_BASE`. And on the vim script side, what needs to be done is to set the environment variable to the correct URL before importing the leetcode module.

Please don't hesitate to tell me your opinions on it.

Reply:
Yes, in my original idea, I was in agreement with you. Because of my poor python skill, I don't know how to implement it..

Reply:
Thanks!

Reply:
This could be helpful: https://stackoverflow.com/questions/5971312/how-to-set-environment-variables-in-python